llama_index/vector_stores/__init__.py

```diff
@@ -8,7 +8,7 @@
 from llama_index.vector_stores.cassandra import CassandraVectorStore
 from llama_index.vector_stores.chatgpt_plugin import ChatGPTRetrievalPluginClient
 from llama_index.vector_stores.chroma import ChromaVectorStore
-from llama_index.vector_stores.cogsearch import CognitiveSearchVectorStore
+from llama_index.vector_stores.azureaisearch import AzureAISearchVectorStore
```

Comment:
We will also need to import CognitiveSearchVectorStore from azureaisearch

Either that, we should probably also update the failing test and make sure it works
`llama_index/tests/vector_stores/test_cogsearch.py`

You can run tests with
`pytests llama_index/tests/vector_stores/test_cogsearch.py`

You might need to do `pip install pytest` for it to work, if it complains about missing pytest